src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (!sourceString.trim()) {
+    return {};
+  }
+
+  return sourceString
+    .split(';')
+    .map((style) => style.trim())
```

Comment:
[CODE KNOWLEDGE] - There's no need to make a copy before each operation. Methods like `map` do not mutate the original array. You can chain the `map` directly after `split` without the need for `trim` on the whole string.